src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol_Bases.cs

```diff
@@ -156,9 +156,12 @@ protected override void CheckInterfaces(DiagnosticBag diagnostics)
 
                             if (other.Equals(@interface, TypeCompareKind.IgnoreNullableModifiersForReferenceTypes))
                             {
-                                diagnostics.Add(ErrorCode.WRN_DuplicateInterfaceWithNullabilityMismatchInBaseList, location, @interface, this);
+                                if (!other.Equals(@interface, TypeCompareKind.ObliviousNullableModifierMatchesAny))
+                                {
+                                    diagnostics.Add(ErrorCode.WRN_DuplicateInterfaceWithNullabilityMismatchInBaseList, location, @interface, this);
```

Comment:
>WRN_DuplicateInterfaceWithNullabilityMismatchInBaseList [](start = 62, length = 55)

From the description of this PR I got an impression that we should report an error now: "we're treating such duplicates (whether direct or indirect) as an error (instead of a warning previously)"

Reply:
Sorry, the compat council went back and forth on this and I forgot to update the description. We're sticking with a warning as opposed to an error because of multi-assembly scenarios.